### ISSUE TITLE
Decode error while emitting redis protocol

### ISSUE THREAD

**Issue by sergunSpb** [sripathikrishnan/redis-rdb-tools#45]
Hello,
i have hash in redis 

```
127.0.0.1:6379> hvals shop:47
1) "{\"data\":{\"cost\":\"4490\",\"country\":\"\xd0\x9a\xd1\x80\xd0\xbe\xd1\x81\xd1\x81\xd0\xbe\xd0\xb2\xd0\xba\xd0\xb8 Zoom Breathe 2K11\",\"pict300\":\"files/images/bonsport/p300_NI068SH95GYY.jpg\",\"discount\":\"0\",\"old_price\":\"4490\",\"pict_mini\":\"files/images/bonsport/m_NI068SH95GYY.jpg\",\"url\":\"asd\",\"pict\":\"files/images/bonsport/p240_NI068SH95GYY.jpg\"},\"id\":\"ABC123\"}"

```

When i try to convert rdb to redis protocol 

```
 sudo rdb --command protocol /var/lib/redis/redis.rdb > /tmp/1
```

, i get error 

```
sudo rdb --command protocol /var/lib/redis/redis.rdb > /tmp/1
Traceback (most recent call last):
  File "/usr/local/bin/rdb", line 9, in <module>
    load_entry_point('rdbtools==0.1.6', 'console_scripts', 'rdb')()
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/cli/rdb.py", line 82, in main
    parser.parse(dump_file)
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/parser.py", line 306, in parse
    self.read_object(f, data_type)
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/parser.py", line 398, in read_object
    self._callback.hset(self._key, field, value)
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/callbacks.py", line 318, in hset
    self.emit('HSET', key, field, value)
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/callbacks.py", line 298, in emit
    self._out.write(u"$" + unicode(len(unicode(arg))) + u"\r\n")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd0 in position 34: ordinal not in range(128)

```

. I don't know python and i don't undestand why this error raise. 
Is it bug ?

**Comment by jlautala** [comment]
It is related to this PR (not sure if it is advancing):
https://github.com/sripathikrishnan/redis-rdb-tools/pull/39

**Comment by oranagra** [comment]
fixed by #77
